eZ/Publish/Core/MVC/Symfony/Cache/Http/FOSPurgeClient.php

```diff
@@ -34,21 +34,28 @@ public function __destruct()
         $this->cacheManager->flush();
     }
 
-    public function purge($locationIds)
+    public function purge($tags)
     {
-        if (empty($locationIds)) {
+        if (empty($tags)) {
             return;
         }
 
-        if (!is_array($locationIds)) {
-            $locationIds = array($locationIds);
+        // As xkey only support one tag (key) being invalidated at a time, we loop.
```

Comment:
To align with [xkey offical example](https://github.com/varnish/varnish-modules/blob/master/docs/vmod_xkey.rst) as is also done in [FOS](https://github.com/FriendsOfSymfony/FOSHttpCache/pull/332/files) and [our VCL PR](https://github.com/ezsystems/ezplatform/pull/143), lets change this to use `key` _(only for Purges, for response headers for tags we use `xkey` as automatically used by the VMOD)_.